Arrangement_on_surface_2/include/CGAL/Arrangement_2/Arrangement_on_surface_2_global.h

```diff
@@ -182,7 +187,7 @@ void insert(Arrangement_on_surface_2<GeomTraits, TopTraits>& arr,
     std::cout << "xxxxxxxxxxxx" << std::endl;
   }
 */
-  arr_zone.init (c, pl);
+  arr_zone.init(c, pl);
```

Comment:
Since you are cleaning the files, any reason to keep the part with the _Needs to be deleted!!!_ flag?